backend/app/controllers/survey_visits_controller.rb

```diff
@@ -5,7 +5,7 @@ class SurveyVisitsController < ApplicationController
 
   # GET /survey_visits or /survey_visits.json
   def index
-    @survey_visits = SurveyVisit.all
+    @survey_visits = SurveyVisit.where(search_params).first(30)
```

Comment:
Unless we think a search is unlikely to return more than 30 results I think it might be worth using [Kaminar](https://github.com/kaminari/kaminari)i to paginate here.

Reply:
Yeah, I wondered about this before – the google doc spec said take 30, but I agree this could be a problem. I think if we're filtering to a specific home that should be fine, but if we want to filter more generally there could be too many results.

Reply:
Okay, have removed the `.first(30)` part – does this otherwise look good to go?